### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by 2roy999**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -49,10 +49,10 @@ process.on('SIGINT', () => {
 
 process.on('uncaughtException', err => {
   logger.fatal(err.message)
-  process.exit(1)
+  setTimeout(() => process.exit(1), 2000)
```
Comment (idanstark42) [inline server/index.js]: Remind why is there a timeout here?

File: test/server/clock.test.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+  it('changes the time according to time', () => {
+    clock.startCountdown(50)
+
+    systemTimer.tick(30 * SECOND)
+
+    expect(clock).property('time').to.be.equal(20)
+  })
+
+  it('changes the time stop changing the time after stopping the countdown', () => {
```
Comment (idanstark42) [inline test/server/clock.test.js]: Dos you mean: " stops changing the time"?

File: server/index.js (hunk 1 above)
Comment (2roy999) [reply server/index.js]: By rethinking about the original problem. we don't need it. It's a sort of a problem in the devl

File: test/server/clock.test.js (hunk 2 above)
Comment (2roy999) [reply test/server/clock.test.js]: Fix it

**Pull request merged**